README.md

```diff
@@ -1 +1,71 @@
 # Vehicle App C++ Template
+
+![Template CI Workflow](https://github.com/eclipse-velocitas/vehicle-app-cpp-template/actions/workflows/ci.yml/badge.svg)
+[![License: Apache](https://img.shields.io/badge/License-Apache-yellow.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+
+The Vehicle App Template for C++ allows to create `Vehicle Apps` from the [Velocitas](https://github.com/eclipse-velocitas/velocitas-docs) development model in the C++ programming language.
+
+## Folder structure
+
+* 📁 `app` - base directory for a vehicle app
+    * 📁 `src` - source code of the vehicle app
+    * 📁 `tests` - tests for the vehicle app
+    * 📁 `vehicle_model` - vehicle model to be used by the vehicle app
+
+## Building
+
+### Installing dependencies
+Before a build can be started, all dependencies required by the App need to be installed. Issue the following command in the SDK root directory:
```

Comment:
It is executed during post-create of the dev container. I moved this to the troubleshooting section of the README in case the automatic execution fails.